PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1107,6 +1107,7 @@ Translation note:
 					<Item id="6311" name="Tab character"/>
 					<Item id="6510" name="Use default value"/>
 					<Item id="6335" name="Treat backslash as escape character for SQL"/>
+					<Item id="6512" name="Backspace key unindents instead of removing single space"/>
```

Comment:
Please don't forget custom_english.xml

Reply:
I've never updated the "custom" version of the file before...but OK, I will do so this time and also moving forward.